src/Infrastructure/Event/RequestTransformerListener.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Event;
+
+use JsonException;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+
+class RequestTransformerListener
+{
+    /**
+     * @var array<string>
+     */
+    protected array $contentTypes;
+
+    /**
+     * @param array<string> $contentTypes
+     */
+    public function __construct(array $contentTypes)
+    {
+        $this->contentTypes = $contentTypes;
+    }
+
+    /**
+     * @param \Symfony\Component\HttpKernel\Event\RequestEvent $event
+     *
+     * @return void
+     */
+    public function onKernelRequest(RequestEvent $event): void
+    {
+        $request = $event->getRequest();
+
+        if ($this->supports($request) === false) {
```

Comment:
it is not necessary to use strict comparison here
Off. doc: `Returns true if needle is found in the array, false otherwise.`